scripts/ceph-windows/collect-event-logs.ps1

```diff
@@ -0,0 +1,45 @@
+param (
+    [Parameter(Mandatory)]
+    [string]$LogDirectory,
+    [switch]$IncludeEvtxFiles = $false,
+    [switch]$CleanupEventLog = $false
+)
+
```

Comment:
After `param` section, please add:
```powershell
$ErrorActionPreference = "Stop"
```

This makes the PowerShell script execution stop, if any exception is thrown.